src/Tools/dotnet-counters/Exporters/ICounterRenderer.cs

```diff
@@ -1,16 +1,18 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Diagnostics.Monitoring.EventPipe;
+
 namespace Microsoft.Diagnostics.Tools.Counters.Exporters
 {
-    public interface ICounterRenderer
+    internal interface ICounterRenderer
     {
-        void Initialize();
-        void EventPipeSourceConnected();
-        void ToggleStatus(bool paused);
+        void Initialize(); //Maps to started?
```

Comment:
Maybe let's clean up these comments: a clear mapping between ICounterLogger methods and ICounterRenderer methods